domain/src/main/java/org/apache/rocketmq/eventbridge/domain/model/connection/ConnectionService.java

```diff
@@ -154,18 +163,12 @@ private AuthParameters setSecretData(AuthParameters authParameters, String accou
         final ApiKeyAuthParameters apiKeyAuthParameters = authParameters.getApiKeyAuthParameters();
         final OAuthParameters oauthParameters = authParameters.getOauthParameters();
         if (basicAuthParameters != null) {
-            Map<String, String> basicAuthParametersMap = Maps.newHashMap();
-            basicAuthParametersMap.put("username", basicAuthParameters.getUsername());
-            basicAuthParametersMap.put("password", basicAuthParameters.getPassword());
-            final String secretName = secretManagerAPI.createSecretName(accountId, connectionName, new Gson().toJson(basicAuthParametersMap));
+            final String secretName = secretManagerAPI.createSecretName(accountId, connectionName, new Gson().toJson(basicAuthParameters));
             basicAuthParameters.setPassword(secretName);
```

Comment:
Why secretName convert to  password? where is the userName?

Reply:
userName itself has value, password just saves the rule name after kms is created.